src/interfaces/IPRBProxyRegistry.sol

```diff
@@ -135,13 +141,14 @@ interface IPRBProxyRegistry {
     function deploy() external returns (IPRBProxy proxy);
 
     /// @notice Deploys a new proxy via CREATE2 by setting the caller as the owner, and delegate calls to the provided
-    /// target contract by forwarding the data. It returns the data it gets back, bubbling up any potential revert.
+    /// target contract by forwarding the data. It returns the data it gets back, and bubbles up any potential revert.
     ///
     /// @dev Emits a {DeployProxy} and an {Execute} event.
     ///
     /// Requirements:
     /// - The caller must not have a proxy.
-    /// - All from {PRBProxy.execute}.
+    /// - `target` must be a contract.
+    /// - `target` must not be the registry.
```

Comment:
`target == registry` is not explicitly checked

Reply:
Good catch.